pkg/controller/scyllacluster/resource.go

```diff
@@ -625,6 +640,103 @@ func MakePodDisruptionBudget(c *scyllav1.ScyllaCluster) *v1beta1.PodDisruptionBu
 	}
 }
 
+func MakeIngresses(c *scyllav1.ScyllaCluster, services map[string]*corev1.Service) []*networkingv1.Ingress {
+	// Don't create Ingresses if cluster isn't exposed.
+	if c.Spec.ExposeOptions == nil {
+		return nil
+	}
+
+	ports := map[string]string{
+		naming.BackendCQL:        naming.PortNameCQLSSL,
+		naming.BackendAlternator: naming.PortNameAlternator,
+	}
+
+	backends := map[string]*scyllav1.IngressOptions{}
+	if c.Spec.ExposeOptions.CQL != nil && c.Spec.ExposeOptions.CQL.Ingress != nil {
+		backends[naming.BackendCQL] = c.Spec.ExposeOptions.CQL.Ingress
+	}
+	if c.Spec.ExposeOptions.Alternator != nil && c.Spec.ExposeOptions.Alternator.Ingress != nil {
+		backends[naming.BackendAlternator] = c.Spec.ExposeOptions.Alternator.Ingress
+	}
+
+	var ingresses []*networkingv1.Ingress
+
+	pathPrefix := networkingv1.PathTypePrefix
+	for backend, ingressOptions := range backends {
+		for _, service := range services {
```

Comment:
alternator would have just 1 backend service, right?

Reply:
no

Reply:
alternator is load balanced service (unlike CQL) why would it need more then one backend service?

Reply:
As mentioned in previous comment, Alternator drivers also can load balance on the client side: https://www.scylladb.com/2021/04/13/load-balancing-in-scylla-alternator/